src/less/main.less

```diff
@@ -0,0 +1,190 @@
+@some-grey: #727272;
+@some-other-grey: #dfdfdf;
+
+.transition-all {
```

Comment:
you put `.transition-all` in a lot of places, can you explain me why? (I have a lot to learn about css yet!!)